libraries/mediaupload/impl/src/main/kotlin/io/element/android/libraries/mediaupload/ImageCompressor.kt

```diff
@@ -98,13 +102,11 @@ class ImageCompressor @Inject constructor(
             is ResizeMode.None -> return
         }
         // Read bounds only
-        inputStream.mark(inputStream.available())
         options.inJustDecodeBounds = true
         BitmapFactory.decodeStream(inputStream, null, options)
         // Set sample size based on the outWidth and outHeight
         options.inSampleSize = options.calculateInSampleSize(width, height)
         // Now read the actual image and rotate it to match its metadata
-        inputStream.reset()
```

Comment:
I was about to ask if this could be removed. 👍  (the comment above is outdated/misplaced, but this is not new)

Reply:
Good catch, I completely missed it! Fixed in https://github.com/vector-im/element-x-android/pull/1501/commits/4eebd8e935e9f71a8cf8b7617069f4433d877dfc